cognitive/src/main/scala/com/microsoft/azure/synapse/ml/cognitive/SpeechSchemas.scala

```diff
@@ -67,3 +71,67 @@ object SpeechSynthesisError extends SparkBindings[SpeechSynthesisError] {
 }
 
 case class SpeechSynthesisError(errorCode: String, errorDetails: String, errorReason: String)
+
+trait HasLocaleCol extends HasServiceParams {
+  val locale = new ServiceParam[String](this,
+    "locale",
+    s"The locale of the input text",
+    isRequired = true)
+
+  def setLocale(v: String): this.type = setScalarParam(locale, v)
+  def setLocaleCol(v: String): this.type = setVectorParam(locale, v)
+}
+
+trait HasTextCol extends HasServiceParams {
+  val text = new ServiceParam[String](this,
+    "text",
+    s"The text to synthesize",
+    isRequired = true)
+
+  def setText(v: String): this.type = setScalarParam(text, v)
+  def setTextCol(v: String): this.type = setVectorParam(text, v)
+}
+
+trait HasVoiceCol extends HasServiceParams {
+  val voice = new ServiceParam[String](this,
+    "voice",
+    s"The name of the voice used for synthesis",
+    isRequired = true)
+
+  def setVoiceName(v: String): this.type = setScalarParam(voice, v)
+  def setVoiceNameCol(v: String): this.type = setVectorParam(voice, v)
+}
+
+trait HasSSMLOutputCol extends Params {
+  val ssmlOutputCol = new Param[String](this, "ssmlCol", "The name of the SSML column")
+
+  def setSSMLOutputCol(value: String): this.type = set(ssmlOutputCol, value)
+
+  def getSSMLOutputCol: String = $(ssmlOutputCol)
+}
+
+trait HasSSMLGeneratorParams extends HasServiceParams
+  with HasLocaleCol with HasTextCol with HasVoiceCol
+  with HasSSMLOutputCol with HasOutputCol with HasErrorCol
+
+case class TextToSpeechSSMLError(errorCode: String, errorDetails: String)
+  object TextToSpeechSSMLError extends SparkBindings[TextToSpeechSSMLError]
+
+case class SSMLConversation(Begin: Int,
+                        End: Int,
+                        Content: String,
+                        Role: String,
+                        Style: String)
```

Comment:
nit: spacing looks a little off here, consider a ctr-alt-L in intelliJ to auto-format

Reply:
Addressed in most recent commit.